pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -338,7 +338,7 @@ func getMarkerScanners(logsCommandName, securityPolicyCommandFormat, setProbeCom
 		func(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 			return deployanalysis.FindDeploymentConfigReadinessWarnings(g, f, setProbeCommandName)
 		},
-		routeanalysis.FindMissingPortMapping,
+		routeanalysis.FindPortMappingIssues,
```

Comment:
All of the names used in marking are similar. If we feel that there is room for improvement we should do it in a separate PR for all of them.